src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -53,16 +59,32 @@ public class IntakeSubsystem extends SubsystemBase {
 					.withSize(1, 1)
 					.getEntry();
 
+	private final GenericEntry getIndexSensorEntry =
+			Shuffleboard.getTab("Intake")
+					.add("Index Sensor - ", getIndexSensor())
```

Comment:
This causes crashes (as well as the `getFeederSensorEntry` below) because the index sensor hasn't been initialized yet. See #29 for a similar error that launcher subsystem ran into, and how to fix it.